source/security/minio-authentication-authorization-review.rst

```diff
@@ -0,0 +1,836 @@
+   <API_PutObjectLegalHold.html>` IAM action.
+
+   This action applies to the following API operations on objects locked
+   under legal hold:
+
+   - ``PutObject``
+
+.. policy-action:: s3:GetBucketObjectLockConfiguration
+   
+   Corresponds to the :s3-api:`s3:GetBucketObjectLockConfiguration
+   <API_GetBucketObjectLockConfiguration.html>` IAM action.
+
+.. policy-action:: s3:PutBucketObjectLockConfiguration
+   
+   Corresponds to the :s3-api:`s3:PutBucketObjectLockConfiguration 
+   <API_PutBucketObjectLockConfiguration.html>` IAM action.
+
+.. policy-action:: s3:GetBucketTagging
+   
+   Corresponds to the :s3-api:`s3:GetBucketTagging <API_GetBucketTagging.html>`
+   IAM action.
+
+.. policy-action:: s3:PutBucketTagging
+   
+   Corresponds to the :s3-api:`s3:PutBucketTagging <API_PutBucketTagging.html>`
+   IAM action.
+
+.. policy-action:: s3:Get
+   
+   Corresponds to the :s3-api:`s3:Get <API_Get.html>` IAM action.
+
+.. policy-action:: s3:Put
+   
+   Corresponds to the :s3-api:`s3:Put <API_Put.html>` IAM action.
+
+.. policy-action:: s3:Delete
+   
+   Corresponds to the :s3-api:`s3:Delete <API_Delete.html>` IAM action.
+
+.. policy-action:: s3:PutBucketEncryption
+   
+   Corresponds to the :s3-api:`s3:PutBucketEncryption
+   <API_PutBucketEncryption.html>` IAM action.
+
+.. policy-action:: s3:GetBucketEncryption
+   
+   Corresponds to the :s3-api:`s3:GetBucketEncryption
+   <API_GetBucketEncryption.html>` IAM action.
+
+.. policy-action:: s3:PutBucketVersioning
+   
+   Corresponds to the :s3-api:`s3:PutBucketVersioning
+   <API_PutBucketVersioning.html>` IAM action.
+
+.. policy-action:: s3:GetBucketVersioning
+   
+   Corresponds to the :s3-api:`s3:GetBucketVersioning
+   <API_GetBucketVersioning.html>` IAM action.
+
+.. policy-action:: s3:GetReplicationConfiguration
+   
+   Corresponds to the :s3-api:`s3:GetReplicationConfiguration 
+   <API_GetReplicationConfiguration.html>` IAM action.
+
+.. policy-action:: s3:PutReplicationConfiguration
+   
+   Corresponds to the :s3-api:`s3:PutReplicationConfiguration
+   <PutReplicationConfiguration.html>` IAM action.
+
+.. policy-action:: s3:ReplicateObject
+   
+   Corresponds to the :s3-api:`s3:ReplicateObject <API_ReplicateObject.html>`
+   IAM action.
+
+.. policy-action:: s3:ReplicateDelete
+   
+   Corresponds to the :s3-api:`s3:ReplicateDelete <API_ReplicateDelete.html>`
+   IAM action.
+
+.. policy-action:: s3:ReplicateTags
+   
+   Corresponds to the :s3-api:`s3:ReplicateTags <API_ReplicateTags.html>` IAM
+   action.
+
+.. policy-action:: s3:GetObjectVersionForReplication
+   
+   Corresponds to the :s3-api:`s3:GetObjectVersionForReplication 
+   <API_GetObjectVersionForReplication.html>` IAM action.
+
+
+.. _minio-auth-authz-pbac-mc-admin-actions:
+
+``mc admin`` Policy Action Keys
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+MinIO supports the following actions for use with defining policies
+for :mc-cmd:`mc admin` operations. These actions are *only* valid for
```

Comment:
might want to bold "only" here (or highlight some other way) italics only with this font is easy to miss

Reply:
Done